pkg/cstor/pvc_operation.go

```diff
@@ -243,3 +255,20 @@ func (p *Plugin) downloadPVC(volumeID, snapName string) (*v1.PersistentVolumeCla
 
 	return pvc, nil
 }
+
+// removePVCAnnotationKey remove the given annotation key from the PVC and update it
+func (p *Plugin) removePVCAnnotationKey(pvc *v1.PersistentVolumeClaim, annotationKey string) error {
+	var err error
+
+	if pvc.Annotations == nil {
+		return nil
+	}
+
+	delete(pvc.Annotations, annotationKey)
```

Comment:
q_: Is it safe to run this, even if the annotationKey is not present on the annotations?

Reply:
yes. https://play.golang.org/p/k0hka1u5CV-